src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -367,32 +364,26 @@ export default {
       } else {
         nextItem = this.filteredItems.find(item => !item.disabled);
       }
-      if (this.hasCta() && this.isCtaAreaFocus) {
```

Comment:
moved below for separation of concerns, here we act on the `nextItem` after we do the `isCtaAreaFocus` blur